packages/aws-cdk-lib/aws-route53/lib/record-set.ts

```diff
@@ -446,12 +446,40 @@ export interface ARecordProps extends RecordSetOptions {
   readonly target: RecordTarget;
 }
 
+/**
+ * Construction properties to import existing ARecord as target.
+ */
+export interface ARecordAttrs extends RecordSetOptions{
+  /**
+   * Existing A record DNS name to set RecordTarget
+   */
+  readonly target: string;
+}
+
 /**
  * A DNS A record
  *
  * @resource AWS::Route53::RecordSet
  */
 export class ARecord extends RecordSet {
+
+  /**
+   * Creates new A record of type alias with target set to an existing A Record DNS.
```

Comment:
```suggestion
   * Creates new Alias Record with the target set to an existing A Record DNS.
```

Reply:
I would still keep the type as A record of type alias to make it more clear as alias record can be of type ARecord or a CName

Reply:
Oh I see, that makes sense. I thought ARecord was just a shortened version of Alias Record.